CHANGELOG.md

```diff
@@ -43,6 +43,15 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
   include the Pods .xcconfig from the base .xcconfig file
   [Rob Hudson](https://github.com/robtimp)
 
+
+## 1.7.0 (2019-05-22)
```

Comment:
this is correct. The two changes below that don't look like part of the diff were already merged on master in a prior PR.

The only new thing master needs is the dSYM stripping change.